props/saber_BC_buttons.h

```diff
@@ -479,16 +540,16 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
       }
     }
     // EVENT_THRUST
-      if (mss.y * mss.y + mss.z * mss.z < 16.0 &&
-          mss.x > 14  &&
-          fusor.swing_speed() < 150) {
-        if (millis() - thrust_begin_millis_ > 15) {
-          Event(BUTTON_NONE, EVENT_THRUST);
-          thrust_begin_millis_ = millis();
-        }
-      } else {
+    if (mss.y * mss.y + mss.z * mss.z < 16.0 &&
```

Comment:
Some of thsee changes seem to be cosmetic, or just re-indenting something.
They slow down reviewing quite a bit, because it's hard to tell if something changed or not.
In the future, try to keep functional and non-functional changes separate.